### PR TITLE
chore(Improve documentation): Environment Variable descriptions

### PR DESCRIPTION
**Pull request by biffgaut**
*Description of changes:*
Include default values in the web site docs
Refer to the environment variables as Lambda function environment variables or container environment variables for clarity.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: 5ac76773ad32ca6901c413c2dbcc64dd908f3c06
* Result: SUCCEEDED
* [Build Logs](https://j8dgcooz5f.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=66de3b87-23f8-4286-b218-6a9e7d756ef5%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v1
* Commit ID: 24790b6752062c3c7440743f105a287340f1277e
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=2deb4f5d-dcca-445b-b866-83a83239b183%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/lib/index.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export interface KinesisstreamsToGluejobProps {
    * The ETL script can be provided either under glue.CfnJob.JobCommandProperty or set as an Asset instance under
    * @KinesisstreamsToGluejobProps.etlCodeAsset.
    *
-   * If an S3 location is know and exists, provide the S3 url in the `scriptLocation` attribute in glue.CfnJob.JobCommandProperty as an
+   * If an S3 location is know and exists, provide the S3URLin the `scriptLocation` attribute in glue.CfnJob.JobCommandProperty as an
```
Comment (mickychetta) [inline source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/lib/index.ts:53]: Add spaces between S3 URL in

File: source/patterns/@aws-solutions-constructs/aws-lambda-sqs-lambda/README.md [hunk 2]
```diff
@@ -107,7 +107,7 @@ new LambdaToSqsToLambda(this, "LambdaToSqsToLambdaPattern", new LambdaToSqsToLam
 |maxReceiveCount?|`number`|The number of times a message can be unsuccessfully dequeued before being moved to the dead letter queue. Defaults to `15`.|
 |existingConsumerLambdaObj?|[`lambda.Function`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-lambda.Function.html)|An optional, existing Lambda function to be used instead of the default function for receiving/consuming messages from the queue. Providing both this and `consumerLambdaFunctionProps` will cause an error. |
 |consumerLambdaFunctionProps?|[`lambda.FunctionProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-lambda.FunctionProps.html)|Optional user-provided properties to override the default properties for the consumer Lambda function.|
-|queueEnvironmentVariableName?|`string`|Optional Name for the SQS queue URL environment variable set for the producer Lambda function.|
+|queueEnvironmentVariableName?|`string`|Optional Name for the Lambda function environment variable set to theURLof the queue. Default: SQS_QUEUE_URL |
```
Comment (mickychetta) [inline source/patterns/@aws-solutions-constructs/aws-lambda-sqs-lambda/README.md:110]: Add spaces between the URL of

**Review by mickychetta — CHANGES_REQUESTED** [review]
Good, just very minor typos

**Review by mickychetta — APPROVED** [review]

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v1
* Commit ID: b848983689399ac4cf854204709507a7e66afb84
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=3756f6bd-6e8d-475a-9862-5e81b2882185%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: 3a9cc1a9558f5d08d41ef851b6ce83c6b567c184
* Result: SUCCEEDED
* [Build Logs](https://j8dgcooz5f.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=f2b243d1-bcb4-4857-a19b-e22323aa70e8%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v1
* Commit ID: 3a9cc1a9558f5d08d41ef851b6ce83c6b567c184
* Result: SUCCEEDED
* [Build Logs](https://nis3u8ftb6.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=85b31439-3e6d-42d5-8226-eb867287c091%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Review by mickychetta — APPROVED** [review]

**Pull request merged**